content/articles/how-to-solve-initial-value-problem-using-ode45-in-matlab/index.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-solve-initial-value-problem-using-ode45-in-matlab/
+title: How to solve initial value problem(IVP) using ODE45 in Matlab
+description: 
+author: joseph_odhiambo
+date: 2021-09-15T00:00:00-18:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  -url: /engineering-education/how-to-solve-initial-value-problem-using-ode45-in-matlab/hero.jpg
+   alt: MATLAB solve Initial Value Problem example image
+---
+
+
+### Introduction
+
+MATLAB’s uses the `ode45` function as the standard solver for ordinary differential equations. The `ode45` function applies Runga-Kutta formulae with the time step variable for easy computation. `ode45` is meant to solve the equation of the general form shown below;
+
+dx/dt = f(t,x), x(t0) = xo
+
+Here, `t` is the independent variable, `x` is a vector of dependent variables to be found, and f (t, x) is a function of t and x. The mathematical equation is defined when the `x` of functions on the right-hand side of Eq. (1), f (t, x), is pre-determined, and the initial conditions, x = x0 at time t0, are given.
```

Comment:
Use Inline latex to denote mathematical terms. Refer to the earlier published article on MATLAB.